ResearchKit/ActiveTasks/ORKRangeOfMotionStepViewController.m

```diff
@@ -44,8 +44,14 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 
 
 #define radiansToDegrees(radians) ((radians) * 180.0 / M_PI)
+
 #define allOrientationsForPitch(x, w, y, z) (atan2(2.0 * (x*w + y*z), 1.0 - 2.0 * (x*x + z*z)))
 
+#define allOrientationsForRoll(x, w, y, z) (atan2(2.0 * (y*w - x*z), 1.0 - 2.0 * (y*y + z*z)))
+
+#define allOrientationsForYaw(x, w, y, z) (asin(2.0 * (x*y + w*z)))
```

Comment:
Could you change all of this defines to inline functions? It's safer.

Reply:
Hi @rsanchezsaez 
Sorry, I'm not sure what you mean... could you make these changes so that I can see... ?
Best wishes

Reply:
Hi @rsanchezsaez, how can we get these requested changes finalised so that this pull request can be approved? Thanks